packages/smarthr-ui/src/components/TextLink/stories/TextLink.stories.tsx

```diff
@@ -0,0 +1,113 @@
+import { action } from '@storybook/addon-actions'
+import React, { ComponentPropsWithoutRef } from 'react'
+
+import { FaCircleQuestionIcon, FaUpRightFromSquareIcon } from '../../Icon'
+import { UpwardLink } from '../../UpwardLink'
+import { TextLink } from '../TextLink'
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+const _prefixOptions = {
+  なし: undefined,
+  あり: <FaCircleQuestionIcon />,
+}
+const _suffixOptions = {
+  なし: undefined,
+  あり: <FaUpRightFromSquareIcon />,
+}
+const _elementAsOptions = {
+  なし: undefined,
+  あり: ({
+    to,
+    ...rest
+  }: Omit<ComponentPropsWithoutRef<'a'>, 'href'> & {
+    to: ComponentPropsWithoutRef<'a'>['href']
+    // eslint-disable-next-line jsx-a11y/anchor-has-content
+  }) => <a {...rest} href={to} />,
+}
+
+export default {
+  title: 'Navigation（ナビゲーション）/TextLink',
+  component: TextLink,
+  subcomponents: { UpwardLink },
+  render: (args) => <TextLink {...args} />,
+  argTypes: {
+    href: {
+      control: 'text',
+    },
+    prefix: {
+      control: 'radio',
+      options: Object.keys(_prefixOptions),
+      mapping: _prefixOptions,
+    },
+    suffix: {
+      control: 'radio',
+      options: Object.keys(_suffixOptions),
+      mapping: _suffixOptions,
+    },
+    elementAs: {
+      control: 'radio',
+      options: Object.keys(_elementAsOptions),
+      mapping: _elementAsOptions,
+    },
+  },
+  args: {
+    href: '#',
+    children: 'リンク',
+  },
+  parameters: {
+    chromatic: { disableSnapshot: true },
+  },
+} as Meta<typeof TextLink>
```

Comment:
今更なんですがsatisfiesでもよさそう
```suggestion
} satisfies Meta<typeof TextLink>
```

Reply:
修正しました! [c2fc910](https://github.com/kufu/smarthr-ui/pull/5033/commits/c2fc9102e92a86d34cae2600aea07fbf69705cc2)
雛形の PR もオープン。 https://github.com/kufu/smarthr-ui/pull/5048